src/ScopeRequest.js

```diff
@@ -102,6 +111,28 @@ class ScopeRequest {
     return true;
   }
 
+  /**
+   * Validate the constraints of an Scope Request
+   * @param filter of an aggregation in the Scope Request
+   * @returns {boolean} true|false
+   */
+  static validateAggregationFilter(filter) {
+    const operatorKeys = _.keys(filter);
+
+    if (operatorKeys.length !== 1) {
```

Comment:
I'm not sure this should always be the case. What about a query with a sort and a limit - wouldn't that be valid? For the others, do we actually need to check this? Sometimes this will not make sense as the operator will be on a single array value but does that matter? i.e. if a pipeline included a max and a min it wouldn't matter as the aggregates can still be applied, even if it doesn't make 'business logic' sense

Reply:
it is valid, added a test for this.

this validates the right side